contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--2.3.0--2.4.0.sql

```diff
@@ -75,5 +75,95 @@ ALTER FUNCTION sys.newid() STABLE;
 ALTER FUNCTION sys.NEWSEQUENTIALID() STABLE;
 ALTER FUNCTION sys.babelfish_typecode_list() STABLE;
 
+alter OPERATOR sys.= (sys.bbf_binary, sys.bbf_binary) 
+set (
+    RESTRICT = eqsel
+);
+
+alter OPERATOR sys.= (sys.bbf_varbinary, sys.bbf_varbinary) 
+set (
+    RESTRICT = eqsel
+);
+
+create or replace function get_bbf_binary_ops_count(opffamily varchar) returns int as $$
+    begin
+        return (select count(*) FROM pg_am am, pg_opfamily opf, pg_amop amop WHERE 
+            opf.opfmethod = am.oid AND amop.amopfamily = opf.oid and opf.opfname = opffamily);
+    end;
+$$ LANGUAGE plpgsql;
+
+DO $$
+    DECLARE bbf_binary_ops_c INT:=(select * from get_bbf_binary_ops_count('bbf_binary_ops'));
+BEGIN
+    IF bbf_binary_ops_c = 5 then
+        CREATE OR REPLACE FUNCTION sys.binary_varbinary_eq(leftarg sys.bbf_binary, rightarg sys.bbf_varbinary)
+        RETURNS boolean
+        AS 'babelfishpg_common', 'varbinary_eq'
+        LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+        CREATE OR REPLACE FUNCTION sys.bbf_binary_varbinary_cmp(sys.bbf_binary, sys.bbf_varbinary)
+        RETURNS int
+        AS 'babelfishpg_common', 'varbinary_cmp'
+        LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+        CREATE OPERATOR sys.= (
+            LEFTARG = sys.bbf_binary,
+            RIGHTARG = sys.bbf_varbinary,
+            FUNCTION = sys.binary_varbinary_eq,
+            COMMUTATOR = =,
+            RESTRICT = eqsel
+        );
+
+        -- PG will create operator family when creating operator class for bbf_binary_ops
+        -- when didn't assign a operator family when creating
+        alter OPERATOR family bbf_binary_ops USING btree add
+            OPERATOR 3 sys.= (sys.bbf_binary, sys.bbf_varbinary),
+            FUNCTION 1 sys.bbf_binary_varbinary_cmp(sys.bbf_binary, sys.bbf_varbinary);
+
+    else if bbf_binary_ops_c = 6 THEN
+        raise notice 'operator of bbf_binary_ops is installed';
+        else 
+            raise exception 'wrong operator numbers in bbf_binary_ops';
+        END IF;
+    END IF;
```

Comment:
1. Indentation
2. Why are there two `END IF`?

Reply:
first end if is for 
IF bbf_varbinary_ops_c = 5 then
else
end if

second end if is for 
if bbf_binary_ops_c = 6 THEN 
else 
endif